charter/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Font and Text Community Group Charter
+---
+This charter:  {{ page.url | remove: "index.html" | prepend: site.url }}
+
+Feedback about this document:
+    https://github.com/font-text/font-text.github.io/issues
+
+## Scope
+
+The Font and Text community group gathers individuals and organisations
```

Comment:
`organisations` should be `organizations` to be consistent with the rest of the document.

Reply:
```suggestion
The Font and Text community group gathers individuals and organizations
```